### PR TITLE
docs: Add a firecracker installation guide

### PR DESCRIPTION
**Pull request by gntouts**
Add info on setting up kata with firecracker.

Fixes: #3555

Signed-off-by: George Ntoutsos <gntouts@nubificus.co.uk>

### CODE REVIEW

**Review by GabyCT — APPROVED** [review]

File: docs/how-to/how-to-use-kata-containers-with-firecracker.md [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+To make the binaries available from the default system `PATH` it is recommended to move them to `/usr/local/bin` or add a symbolic link:
+
+```bash
+$ sudo ln -s $(pwd)/firecracker /usr/local/bin
+$ sudo ln -s $(pwd)/jailer /usr/local/bin
+```
+
+More details can be found in [AWS Firecracker docs](https://github.com/firecracker-microvm/firecracker/blob/main/docs/getting-started.md)
+
+In order to run Kata with AWS Firecracker a block device as the backing store for a VM is required. To interact with `containerd` and Kata we use the `devmapper` `snapshotter`.
```
Comment (amshinde) [inline docs/how-to/how-to-use-kata-containers-with-firecracker.md:43]: Can you also mention that the script below would require 10G of free space on disk.

File: docs/how-to/README.md [hunk 2]
```diff
@@ -37,4 +42,4 @@
 - [How to setup swap devices in guest kernel](how-to-setup-swap-devices-in-guest-kernel.md)
 - [How to run rootless vmm](how-to-run-rootless-vmm.md)
 - [How to run Docker with Kata Containers](how-to-run-docker-with-kata.md)
-- [How to run Kata Containers with `nydus`](how-to-use-virtio-fs-nydus-with-kata.md)
\ No newline at end of file
+- [How to run Kata Containers with `nydus`](how-to-use-virtio-fs-nydus-with-kata.md)
```
Comment (amshinde) [inline docs/how-to/README.md:45]: Is this a desired change?

**Review by amshinde — COMMENTED** [review]
@gntouts Thanks for adding this document. Looks good to me, just a minor comment from my side.

File: docs/how-to/README.md (hunk 2 above)
Comment (gntouts) [reply docs/how-to/README.md]: We removed this change. Thanks for pointing it out.

**Review by amshinde — APPROVED** [review]

**Pull request merged**